src/Network/WebSockets/Http.hs

```diff
@@ -238,6 +239,11 @@ getRequestSubprotocols rh = maybe [] parse mproto
         mproto = lookup "Sec-WebSocket-Protocol" $ requestHeaders rh
         parse = filter (not . B.null) . BC.splitWith (\o -> o == ',' || o == ' ')
 
+--------------------------------------------------------------------------------
+-- | Get the @Sec-WebSocket-Extensions@ header
+getRequestSecWebSocketExtensions :: RequestHead -> Maybe B.ByteString
+getRequestSecWebSocketExtensions p =
+    lookup "Sec-WebSocket-Extensions" (requestHeaders p)
```

Comment:
I will use your suggestion, but there is only one extension. It is not clear (to me) how to negotiate multiple extensions and multiple configs each. The above was just to get the header value.